dev.zed.Zed.Preview.yaml

```diff
@@ -0,0 +1,210 @@
+          type: json
+          url: https://api.github.com/repos/zed-industries/zed/releases
+          version-query: map(select(.prerelease)) | first | .tag_name
+          url-query:
+            map(select(.prerelease)) | first | .assets[] | select(.name=="zed-linux-x86_64.tar.gz")
+            | .browser_download_url
+          is-main-source: true
+      - type: archive
+        url: https://github.com/zed-industries/zed/releases/download/v0.184.4-pre/zed-linux-aarch64.tar.gz
+        sha256: fdaf297297bdcc12859364d2f0303666565d7d2b818dd738e63cf1373f227178
+        only-arches:
+          - aarch64
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/zed-industries/zed/releases
+          version-query: map(select(.prerelease)) | first | .tag_name
+          url-query:
+            map(select(.prerelease)) | first | .assets[] | select(.name=="zed-linux-aarch64.tar.gz")
+            | .browser_download_url
+          is-main-source: true
+      - type: file
+        path: dev.zed.Zed.Preview.metainfo.xml
+
+  - name: host-spawn
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 host-spawn /app/bin/host-spawn
+    sources:
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.1/host-spawn-x86_64
+        sha256: 733746ab498e07d065cbecf80bacd447eb21846d1462e8fe23fdd9d9dc977b50
+        dest-filename: host-spawn
+        only-arches: [x86_64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/1player/host-spawn/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test(".+x86_64$"))][0] | .browser_download_url'
+      - type: file
+        url: https://github.com/1player/host-spawn/releases/download/v1.6.1/host-spawn-aarch64
+        sha256: 71b7744a4d0b29279523cc0f0ed1a7af5e9555510bd9e6d4685391b59faefaac
+        dest-filename: host-spawn
+        only-arches: [aarch64]
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/1player/host-spawn/releases/latest
+          tag-query: .tag_name
+          version-query: .tag_name
+          timestamp-query: .published_at
+          url-query: '[.assets[] | select(.name|test(".+aarch64$"))][0] | .browser_download_url'
+
+  - name: zed-wrapper
+    buildsystem: meson
+    config-opts:
+      - -Deditor_binary=/app/bin/zed
+      - -Deditor_title=Zed
+      - -Dprogram_name=zed-wrapper
+      - -Dfirst_run_template=README.md
+      - -Dsdk_update_template=sdk_update.md
+      - -Dflagfile_prefix=flatpak-zed
+      - -Ddefault_loglevel=0
+    sources:
+      - type: git
+        commit: 3314b55ccd4ece8810715787e246f8407e20caab
+        url: https://github.com/flathub-infra/ide-flatpak-wrapper
+      - type: file
+        path: README.md
+      - type: file
+        path: sdk_update.md
```

Comment:
neither files exist here.

Reply:
I will add `sdk_update.md`. `README.md` is here, no...